src/transformers/integrations/ggml.py

```diff
@@ -690,9 +725,15 @@ def converted(self) -> Tokenizer:
         return tokenizer
 
 
+class GGUFQwen2MoeConverter(GGUFQwen2Converter):
+    def __init__(self, tokenizer_dict):
+        super().__init__(tokenizer_dict)
+
+
```

Comment:
Qwen2Moe and Qwen2 share the same tokenizer, we can just pass `GGUFQwen2Converter` in  `GGUF_TO_FAST_CONVERTERS`

Reply:
Sure, will do